_config.yml

```diff
@@ -69,94 +69,10 @@ authors:
     byline: Core Contributor
     github: tomwfox
     twitter: T0mfox
+  dblythy:
+    name: Daniel Blyth
+    github: dblythy
+    byline: Parse Contributor
```

Comment:
I added your GH and a byline. You can add a twitter handle and website link if you want, but totally up to you.